### PR TITLE
*: a vote for AndrewDi as the committer

### PR DESCRIPTION
**Pull request by bb7133**
As described in the title, thanks.

### CODE REVIEW

File: votes/0547-andrewdi-as-tidb-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* His contributions covered a varity of modules of TiDB-Server(expression, executor, planner, DDL...)
+* He is the one who introduce the 'View' support to TiDB, which is valuable for TiDB as a database.
+
+## Deadline
+
+The vote will be open for at least 2 days unless there is an objection or not enough votes.
+
+## Scope
+
+* TOC
```
Comment (tisonkun) [inline votes/0547-andrewdi-as-tidb-reviewer.md:23]:
```suggestion
* Team TiDB
```

Comment (bb7133) [reply votes/0547-andrewdi-as-tidb-reviewer.md]: Addressed, thanks.

File: votes/0547-andrewdi-as-tidb-reviewer.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* [Reviewed pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%3AAndrewDi)
+
+I (@bb7133) hereby nominate @AndrewDi as TiDB committer and call for a vote for the following reasons:
+
+* His contributions covered a varity of modules of TiDB-Server(expression, executor, planner, DDL...)
+* He is the one who introduce the 'View' support to TiDB, which is valuable for TiDB as a database.
+
+## Deadline
+
+The vote will be open for at least 2 days unless there is an objection or not enough votes.
```
Comment (tisonkun) [inline votes/0547-andrewdi-as-tidb-reviewer.md:19]:
```suggestion
The vote will be open for at least 6 days unless there is an objection or not enough votes.
```

according to https://github.com/pingcap/community/blob/master/teams/README.md#actions

**Comment by disksing** [comment]
cheers!

**Comment by tisonkun** [comment]
> cheers!

@disksing this vote is still under process. You can show your approval by explicitly approving this PR.

**Review by disksing — APPROVED** [review]

File: votes/0547-andrewdi-as-tidb-reviewer.md (hunk 2 above)
Comment (bb7133) [reply votes/0547-andrewdi-as-tidb-reviewer.md]: Fixed, sorry for the mistake!

**Review by winkyao — APPROVED** [review]
LGTM, congratulation to AndrewDi

File: votes/0547-andrewdi-as-tidb-reviewer.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team TiDB
+
+## Result
+
+Conclude the voting result, including approvals and vetoes, binding and non-binding.
```
Comment (tisonkun) [inline votes/0547-andrewdi-as-tidb-reviewer.md:27]: @bb7133 I think this vote can be concluded with result updated.

Comment (bb7133) [reply votes/0547-andrewdi-as-tidb-reviewer.md]: Updated, thanks.

**Comment by bb7133** [comment]
Hi  @pingcap/tidb-maintainers,  3 '+1 votes' have been collected, according to the [current rule](https://github.com/pingcap/community/tree/master/teams), I will merge it tomorrow if there's no new '-1 vote'.

**Pull request merged**